engine/apps/alerts/migrations/0070_remove_resolutionnoteslackmessage__slack_channel_id_db.py

```diff
@@ -12,10 +12,14 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        linter.IgnoreMigration(),
-        common.migrations.remove_field.RemoveFieldDB(
-            model_name='resolutionnoteslackmessage',
-            name='_slack_channel_id',
-            remove_state_migration=('alerts', '0068_remove_resolutionnoteslackmessage__slack_channel_id_state'),
-        ),
+        # NOTE: commented out due to some issues this was causing w/ SQLite:
+        # https://github.com/grafana/oncall/issues/5306
+        # https://github.com/grafana/oncall/issues/5244#issuecomment-2503999986
+        #
+        # linter.IgnoreMigration(),
+        # common.migrations.remove_field.RemoveFieldDB(
+        #     model_name='resolutionnoteslackmessage',
+        #     name='_slack_channel_id',
+        #     remove_state_migration=('alerts', '0068_remove_resolutionnoteslackmessage__slack_channel_id_state'),
+        # ),
```

Comment:
honestly I don't think there is any harm in commenting this out. It's really only doing a `DROP COLUMN` statement. 

For whatever reason, this seems to fail (only on SQLite; see https://github.com/grafana/oncall/issues/5306 for more details):
```
oncall_db_migration-1  | django.db.utils.OperationalError: error in index alerts_reso_ts_a9bdf7_idx after drop column: no such column: _slack_channel_id
oncall_db_migration-1  |   Applying alerts.0070_remove_resolutionnoteslackmessage__slack_channel_id_db...
```